### DISCUSSION TITLE
Unexpected search behaviour

### DISCUSSION THREAD

**Discussion by thisIsTheFoxe** [community/community#24059] [Other Feature Feedback, Questions, & Ideas]
<p>When opening the palette in a repo, and selecting “Issues” or “Pull requests” etc. (using the v and &gt; arrow keys), and then typing something afterward it always shows “No search results”.<br>
Furthermore, the first suggestion, which is to search the repo includes the “issues” or “Pull request” string.</p>
<p><strong>Expected</strong>:<br>
It should show search results without the <em>prefix</em>. It should also ideally match the selected type and show either only PRs or issues etc., depending on what was selected.</p>

**Comment by gaknoll** [comment, marked as answer]
<p><a class="mention" href="/thisisthefoxe">@thisIsTheFoxe</a> thanks for taking the time to share your feedback! We will soon be working on a next iteration of the search experience that does not require using “modes” to search. Based on the scenario you described this should help!</p>
